server/grpc.go

```diff
@@ -82,7 +86,7 @@ type StreamServer struct {
 	errorLogger    *zap.Logger
 }
 
-func NewStreamServer(opt Option) *StreamServer {
+func NewStreamServer(opt Option) (*StreamServer, error) {
```

Comment:
__[golint]__
* [golint] exported function NewStreamServer should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->